index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Kyle Hartman's Portfolio</title>
+    <style>
+        body {background-color: rgb(8, 5, 28);}
+        h1 {color: rgb(175, 152, 20);}
+        p {color: rgb(23, 96, 64);}
+    </style>
+</head>
+
+<body>
+    <h1>KYLE HARTMAN</h1>
+    <section id="About">
+    <h2>About</h2>
+    <p>I am a student at Code The Dream and I am hoping to learn
+        a new career. It is very exciting to see the inner workings
+        of how things are done behind the scenes.</p>
+    </section>
+
+    <section id="Experience">
+    <h2>Experience</h2>
+    <ul>
+        <li>Working in HHS troubleshooting errors</li>
+        <li>Photoshop and editing for a discord server</li>
+        <li>Code the Dream classes</li>
+    </ul>
+    </section>
+
+    <section id="Connect">
+    <h2>Connect</h2>
+    <ul>
+        <il>https://www.linkedin.com/in/kyle-hartman-669823331/</il>
+        <li>https://github.com/khartman88/Jupiter-CTD-Kyle</li>
+    </ul>
+    </section>
+
+</body>
+</html>
```

Comment:
If you want this red icon to go away, this is just GitHub "yelling at you" about not having an extra empty line at the end of each file. That's what "good form" is and what your jobs will have you do.

<img width="29" alt="Screenshot 2024-12-15 at 12 35 51 PM" src="https://github.com/user-attachments/assets/7d86dfd7-4bbb-4a8c-a708-a8d8f8ade4e3" />